bin/hubot.cmd

```diff
@@ -0,0 +1,7 @@
+@echo off
+
+call npm install
+SETLOCAL
+SET PATH=node_modules\.bin;node_modules\hubot\node_modules\.bin;%PATH%
```

Comment:
Is this file for Windows? We wouldn't need that if we're launching in Heroku or similar.

Reply:
Yeah, for window. I will delete it.